### PR TITLE
Entry/Edits: Bootstrap and `void` functions

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

This is part of our team's ongoing work with targeted keywords.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/bootstrap/bootstrap.md [hunk 1]
```diff
@@ -14,18 +14,69 @@ Tags:
...
+  integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+  crossorigin="anonymous"
+></script>
+```
+
+## Themes
+
+As its popularity grew, people started creating templates based on Bootstrap in order to accelerate the web development process even further. There are many websites out there dedicated to sharing and buying custom templates based on Bootstrap.
+
+Here are the official themes built by or reviewed by Bootstrap's creators: https://themes.getbootstrap.com
```
Comment (KTom101) [inline content/general/concepts/bootstrap/bootstrap.md:75]:
```suggestion
Here are the official themes built or reviewed by Bootstrap's creators: https://themes.getbootstrap.com
```

File: content/javascript/concepts/functions/functions.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Functions'
-Description: 'Functions are one of the fundamental building blocks in JavaScript. A function is a reusable set of statements to perform a task or calculate a value. Functions can be passed one or more values and can return a value at the end of their execution. In order to use a function, you must define it somewhere in the scope where you wish to call it. Function declarations are used to create named functions. These functions can be called using their declared name. Function declarations are built from: - The function keyword. - The function name. - An optional list of parameters separated by commas enclosed by a set of parentheses (). - A function body enclosed in a set of curly braces {}. The example code provided contains a function named add() that takes in 2 values and prints the sum of those numbers:'
+Description: 'Functions are reusable pieces of code that perform tasks, process data, and possible return values.'
```
Comment (KTom101) [inline content/javascript/concepts/functions/functions.md:3]: Were you trying to say "possibly" here? 
```suggestion
Description: 'Functions are reusable pieces of code that perform tasks, process data, and possibly return values.'
```

File: content/general/concepts/void/void.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (KTom101) [inline content/general/concepts/void/void.md:1]: This entry should be linked out in the [general data types entry ](https://www.codecademy.com/resources/docs/general/data-types). 
Void is in the "Other Common Data Types" table.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Dusch4593 Here are a few suggestions.

File: content/general/concepts/void/void.md (hunk 3 above)
Comment (Dusch4593) [reply content/general/concepts/void/void.md:1]: Great call! 🙏🏻

**Comment by Dusch4593** [comment]
@KTom101 I'm gonna implement your remaining suggestions locally and push from there 😄

**Comment by Dusch4593** [comment]
@KTom101 I believe I implemented the rest of your suggestions! 👍🏻

**Review by KTom101 — APPROVED** [review]
@Dusch4593 Looks good! 👍

**Review by SSwiniarski — APPROVED** [review]
I didn't see any changes I'd make. LGTM.

**Comment by Dusch4593** [comment]
@KTom101 @SSwiniarski thank you for the feedback! Before we merge, was there anything left to add or check? cc: @yangc95

**Comment by KTom101** [comment]
@Dusch4593  There's a small typo on line 108 in functions.md. It should be,

 "A function is also capable of returning a value back to where it was called using the `return` keyword." 

Currently, "were" is written instead of "where"

**Comment by KTom101** [comment]
@Dusch4593  There's a small typo on line 108 in functions.md. It should be,

 "A function is also capable of returning a value back to where it was called using the `return` keyword." 

Currently, "were" is written instead of "where"

**Pull request merged**